tools/RandomCodeGenerator.js

```diff
@@ -69,6 +75,11 @@ export function generateRandomCode(language, lines) {
       fillerLineQty = parseInt(lines, 10) - 3;
 
       fillerLines.push("{");
+
+      if (addComment) {
```

Comment:
Should we alter the fillerLineQty if the addComment is true? Otherwise, we'll be getting the wrong number of lines.